substrate-node/pallets/pallet-smart-contract/src/lib.rs

```diff
@@ -1137,7 +1138,7 @@ impl<T: Config> Pallet<T> {
         // Clean up contract from billing loop if it not exists anymore
         if Contracts::<T>::get(contract_id).is_none() {
             log::debug!("cleaning up deleted contract from storage");
-            Self::remove_contract_from_billing_loop(contract_id);
+            Self::remove_contract_from_billing_loop(contract_id)?;
```

Comment:
`return Self::remove_contract_from_billing_loop(contract_id)?;`

Reply:
hmm, not compiling
because `?` returns the `Ok()`'s unwrapped  value which would be just `()` in our case